.github/workflows/ci.yml

```diff
@@ -127,6 +127,7 @@ jobs:
           _make PLATFORM=vexpress-qemu_armv8a CFG_NS_VIRTUALIZATION=y
           _make PLATFORM=vexpress-qemu_armv8a CFG_CORE_PREALLOC_EL0_TBLS=y
           _make PLATFORM=vexpress-qemu_armv8a CFG_CORE_SEL1_SPMC=y
+          _make PLATFORM=vexpress-qemu_armv8a CFG_CORE_SEL1_SPMC=y CFG_VIRTUALIZATION=y
```

Comment:
This is now called `CFG_NS_VIRTUALIZATION`.

Reply:
Thanks, I'll fix.